voluseg/_tools/load_metadata.py

```diff
@@ -1,9 +1,29 @@
-def load_metadata(parameters, filename_channel, filename_stack):
-    """fetch z-resolution, exposure time, and stack frequency"""
+import numpy as np
+from xml.etree import ElementTree
 
-    import numpy as np
-    from xml.etree import ElementTree
 
+def load_metadata(
+    parameters: dict,
+    filename_channel: str,
+    filename_stack: str,
+) -> dict:
+    """
+    Fetch z-resolution, exposure time, and stack frequency.
+
+    Parameters
+    ----------
+    parameters : dict
+        Parameters dictionary.
+    filename_channel : str
+        Filename XML file containing channel information.
```

Comment:
this is very specific to janelia so may be removed (or should be generalized).

Reply:
this argument is being used by the function, how should we remove it?

Reply:
Yes, you are right. The full function `load_metadata.py` is specific to janelia. However, since it does not interfere with anything, let's keep it in for compatibility purposes.